ansible/roles/omero-scripts-dependencies/README.tmd

```diff
@@ -0,0 +1,10 @@
+OMERO Scripts Dependencies
+==========================
+
+Install the dependencies required by the OMERO scripts
+
+
```

Comment:
Can you add a note that this role is still incomplete for CentOS7? Also the file extension is `.tmd`, shouldn't it be `.md`?